GoogleMeetup/src/main/webapp/script.js

```diff
@@ -37,8 +48,12 @@ async function createHeatmap() {
     var heatmapData = [];
     var markers = [];
 
-    const map = new google.maps.Map(document.getElementById('map'), {
-        center: new google.maps.LatLng(-33.8, 151.1),
```

Comment:
Do you need this, or can you let it select the zoom location dynamically?

Reply:
Yeah, you need to specify a center initially for the map to load in.